lightningd/plugin_control.c

```diff
@@ -256,6 +257,10 @@ static struct command_result *json_plugin_control(struct command *cmd,
 					json_get_member(buffer, mod_params,
 							"plugin") - 1, 1);
 		}
+		if (access(plugin_path, X_OK) != 0)
+			plugin_path = path_join(cmd,
+					path_join(cmd, cmd->ld->config_basedir, "plugins/"),
+					plugin_path);
```

Comment:
`cmd->ld->plugins->default_dir` is the right thing to use here I think?  

And of course, documenting this in `doc/lightning-plugin.7.md` would be great too!

Reply:
Thanks @rustyrussell . I didn't know `cmd->ld->plugins->default_dir` was available. And yes I should have updated the documentation as well. These are both done now. See commit [2181627]
(https://github.com/ElementsProject/lightning/pull/5211/commits/2181627377ecc310ef026f201ec4968d246c79ee).